proca/lib/proca_web/schema/target_types.ex

```diff
@@ -0,0 +1,45 @@
+defmodule ProcaWeb.Schema.TargetTypes do
+  @moduledoc """
+  Schema decleration for Target and TargetEmail types
+  """
+
+  use Absinthe.Schema.Notation
+  alias ProcaWeb.Resolvers
+  alias ProcaWeb.Resolvers.Authorized
+
+  input_object :target_email do
+    field :email, non_null(:string)
+  end
+
+  input_object :target_input do
+    field :name, non_null(:string)
+    field :area, :string
+    field :campaign_id, non_null(:integer)
+    field :external_id, non_null(:string)
+    field :fields, :json
+    field :emails, list_of(:target_email)
+  end
+
+  object :target do
+    field :id, non_null(:string)
+    field :name, non_null(:string)
+    field :area, :string
+    field :fields, :json
+  end
+
+  object :target_mutations do
+    field :upsert_targets, type: non_null(list_of(:target)) do
```

Comment:
Can we add campaign_id as to argument. so it's not repeated in every target

Reply:
This allows to maintain 1 airtable per org, and have multiple targets in the same sheet with the campaign id as a parameter. It's what we said we'd start from during the call.

Reply:
Hmmm wouldn't having one airtable per campaign be better ? It's more self contained and campaigners cannot break one campaign by editing targets to another.
I remember we talked about being ok with duplication of targets (so for new campaign they will just clone the airtable, and there will be duplication but this is fine)

Reply:
Changed